hudi-common/src/main/java/org/apache/hudi/common/util/collection/DiskBasedMap.java

```diff
@@ -223,8 +223,8 @@ private R get(ValueMetadata entry) {
   private synchronized R put(T key, R value, boolean flush) {
     try {
       byte[] val = SerializationUtils.serialize(value);
-      Integer valueSize = val.length;
-      Long timestamp = System.currentTimeMillis();
+      int valueSize = val.length;
```

Comment:
why change these?

Reply:
The type of `val.lenght` is primitive type(int), the type of `System.currentTimeMillis()` is primitive type(long).

Reply:
Will revert it if this against contribution guide. Thanks

Reply:
Done.